docs/development.md

```diff
@@ -63,6 +63,25 @@ bundle exec foreman start
 
 When using the dev server on Firefox you need to accept the self-signed certificate. You should go to `https://centos7-devel.example.com:3808` (or your equivalent) and add an exception for this certificate. Chrome does not throw any warnings if you have accepted the same certificate already for `https://centos7-devel.example.com`.
 
+### Reviewing Pull Requests
+
+It's easy to checkout pull requests from projects that were installed in development environment. All projects are cloned in vagrant's home, e.g. /home/vagrant/foreman, /home/vagrant/katello etc. In order to apply some PR to any of this project, you can use a reviewer script. See following example
+
+```sh
+cd /home/vagrant/katello
+rpr 5266
+```
+
+`rpr` is shortcut for review pull reqest. This fetches information about PR number 5266, defines new remote in you .git/config, fetches new objects from it. Then it creates new local branch called review/pr5266 and pulls the code from the PR. Don't forget to migration and seed the database if the PR contains related changes. Note that this might create new merge commit so git might want you to set your email and name. You can set that with following commands.
+
+```sh
+git config --global user.email "you@example.com"
+git config --global user.name "Your Name"
+```
+
+Once reviewing is finished, the repository can be reset to develop/master branch by calling `rrpr`. It destroys the review branch after it checkouts back to master branch.
```

Comment:
Is the double 'r' a typo here?

Reply:
no, rrpr is symlink to rpr, the script behaves differently based on which name you use, rrpr stands for reverse review pull request :-) there's one more mode - `crp` which is not documented here since it's only used for final merging, not reviewing